### ISSUE TITLE
Two custombox close method

### ISSUE THREAD

**Issue by marwen-cherif** [dixso/custombox#77]
When we have two custombox opened, is it possible to close a specefic one ?
because Custombox.Close() is generic and for now it close my last one opened.

**Comment by AdKMorris** [comment]
For those that google this like I did the version as of this post can does this in pretty much the same way:
```
myCustombox = new Custombox.modal({
    target: '#modal',
    id: 'myCustom123'
});
```

Then to close:
`Custombox.modal.close('myCustom123');
`
Have not tested with 'first' and 'last' but I assume they will work in the current version as well.

**Comment by spaceknight** [comment]
I spent almost half a day trying to figure out how to close the modal!

I still cannot close it.

I realized that there is an old site which still has the old documentation.

Why such a poor documentation?

@AdKMorris,

your suggestion does not work either. I get the "Error: Custombox is not instantiated" error in console.

we should be able to close it this way but it does not work either:

myCustombox.close();

I get the "ReferenceError: Can't find variable: myCustombox" error.

I am about to give up.
